src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -128,16 +136,14 @@ export default class GroupComparisonMutationsTabPlot extends React.Component<
             this.mutationMapperToolStore.mutationMapperStores,
             this.props.store.coverageInformation,
             this.props.store.groupToProfiledPatients,
+            this.props.store.groupToProfiledPatientCounts,
         ],
         render: () => {
-            if (
-                this.mutationMapperToolStore.getMutationMapperStore(
-                    this.props.store.activeMutationMapperGene!.hugoGeneSymbol
-                )
-            ) {
-                const mutationMapperStore = this.mutationMapperToolStore.getMutationMapperStore(
-                    this.props.store.activeMutationMapperGene!.hugoGeneSymbol
-                );
+            const mutationMapperStore = this.mutationMapperToolStore.getMutationMapperStore(
+                this.props.store.activeMutationMapperGene!.hugoGeneSymbol
+            );
+            if (mutationMapperStore) {
+                mutationMapperStore.uniqueSampleKeyToTumorType = this.props.store.uniqueSampleKeyToTumorType.result!;
```

Comment:
this seems weird that we are taking this from one store and giving it to another like this.  can the logic not make use of uniqueSampleKeyToTumorType on ```this.props.store```?   Also, should ```uniqueSampleKeyToTumorType``` be awaited above?